### PR TITLE
[BEAM-1999] Remove _unused_options_id

### PR DESCRIPTION
**Pull request by mariapython**

### CODE REVIEW

File: sdks/python/apache_beam/runners/direct/direct_runner.py [hunk 1]
```diff
@@ -87,11 +87,7 @@ def run(self, pipeline):
                         evaluation_context)
     # Start the executor. This is a non-blocking call, it will start the
     # execution in background threads and return.
-
-    if pipeline.options:
-      # DirectRunner does not support RuntimeValueProviders.
-      RuntimeValueProvider.set_runtime_options(None, {})
-
+    RuntimeValueProvider.set_runtime_options({})
```
Comment (aaltay) [inline sdks/python/apache_beam/runners/direct/direct_runner.py]: Could you keep the comment `# DirectRunner does not support RuntimeValueProviders.`

Comment (mariapython) [inline sdks/python/apache_beam/runners/direct/direct_runner.py]: Done

**Comment by mariapython** [comment]
@aaltay: PTAL

**Comment by aaltay** [comment]
Run Python PostCommit

**Comment by aaltay** [comment]
LGTM

**Pull request closed without merging**